force-app/lwc/signals/core.js

```diff
@@ -10,6 +10,10 @@ const UNSET = Symbol("UNSET");
 const COMPUTING = Symbol("COMPUTING");
 const ERRORED = Symbol("ERRORED");
 const READY = Symbol("READY");
+const defaultEffectProps = {
+  _fromComputed: false,
+  identifier: null
```

Comment:
this probably should be auto generated by the library to improve log experience, but can be overridden by the end user for a human friendly name.

Reply:
Ok, let me refactor this so it could either be a string or a `Symbol`, so this is a unique symbol by default. That way I can avoid depending on an Id generator